gbpservice/nfp/service_plugins/firewall/nfp_fwaas_plugin.py

```diff
@@ -198,11 +207,33 @@ def _find_routers_via_routes_for_floatingip(self, context, internal_port,
 
     return [p_r[1] for p_r in sorted(prefix_routers, reverse=True)]
 
+def elevate_context(context):
+    context = context.elevated()
+    context.tenant_id = _resource_owner_tenant_id()
+    return context
```

Comment:
return value not used? Are we overwriting original context? Do we need to save and restore it?